apps/version.c

```diff
@@ -157,8 +151,12 @@ int version_main(int argc, char **argv)
         printf("%s\n", OpenSSL_version(OPENSSL_ENGINES_DIR));
     if (moddir)
         printf("%s\n", OpenSSL_version(OPENSSL_MODULES_DIR));
-    if (seed)
-        printf("Seeding source: %s\n", OPENSSL_info(OPENSSL_INFO_SEED_SOURCE));
+    if (seed) {
+        const char *src = OPENSSL_info(OPENSSL_INFO_SEED_SOURCE);
+        printf("Seeding source: %s\n", src ? src : "N/A");
```

Comment:
Hmmm, is that actually possible? No harm, of course, but I thought that it couldn't be configured to nothing...

Reply:
you could link against a libcrypto that does not implement OPENSSL_INFO_SEED_SOURCE
in theory, then the return value could be NULL, and some runtime libs, like
newlib crash on printf("%s", NULL), some don't and print "(null)" or something.

Reply:
Ah, oh, good point, generally speaking

Reply:
explicit "!= NULL" test?

Reply:
This was merged 7 days ago...

Reply:
yes, but perhaps pointing out the missed change will be useful.